packages/server/graphql/mutations/helpers/notifications/notifySlack.ts

```diff
@@ -112,7 +112,7 @@ export const startSlackMeeting = async (
   const title = 'Meeting started :wave: '
   const blocks = [
     makeSection(title),
-    makeSections([`*Team:*\n${team.name}`, `*Meeting:*\n${meeting.name}`]),
+    makeSections([`*Team:*\n${team?.name}`, `*Meeting:*\n${meeting.name}`]),
```

Comment:
-1 If a team is `undefined` this will result in showing *Team*: undefined. Not a big deal, but IMO it's better to not send the notification if something went wrong than to show `undefined` to the users. I'd add a check to make sure the team is defined